app/domain/output_document.rb

```diff
@@ -21,7 +21,7 @@ def attendee_name
   end
 
   def appointment_date
-    appointment_summary.date_of_appointment.to_s(:long)
+    appointment_summary.date_of_appointment.strftime('%e %B %Y')
```

Comment:
I'll declare it in the domain object for now, it can be moved if we need to generate dates in other places later down the line